packages/remix/README.md

```diff
@@ -0,0 +1,273 @@
+# @supabase/auth-helpers-remix
+
+This submodule provides convenience helpers for implementing user authentication in Remix applications.
+
+## Installation
+
+Using [npm](https://npmjs.org):
+
+```sh
+npm install @supabase/auth-helpers-remix
+```
+
+This library supports the following tooling versions:
+
+- Node.js: `^10.13.0 || >=12.0.0`
+
+- Remix: `>=1.5.1`
+
+## Getting Started
+
+### Configuration
+
+Set up the following env vars. For local development you can set them in a `.env` file. See an example [here](../../examples/remix/.env.example)).
+
+```bash
+# Find these in your Supabase project settings > API
+SUPABASE_URL=https://your-project.supabase.co
+SUPABASE_ANON_KEY=your-anon-key
+```
+
+### Basic Setup
+
+<!-- TODO! Come back to here! -->
+
+- Create a route for `/api/auth`
+
+- Create a `[...supabase].js` file under the newly created `auth` directory.
+
+The path to your dynamic API route file would be `/pages/api/auth/[...supabase].js`. Populate that file as follows:
+
+```js
+import { handleAuth } from '@supabase/auth-helpers-nextjs';
```

Comment:
I'm presuming the import needs to come from `@supabase/auth-helps-remix` instead.